app/controllers/mfa_confirmation_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
   private
 
   def enforce_second_mfa?
-    IdentityConfig.store.select_multiple_mfa_options &&
+    IdentityConfig.store.kantara_2fa_phone_restricted &&
```

Comment:
I forget what the difference between these two feature flags is, can you remind me?

Reply:
Yup, Sorry 1. Is for multi selecting MFA options on account creation. 
The other is restricting newly created users from selecting just phone on account creation. I hope that once things become stable and we have permanent green light on it, we can have a cleanup PR removing all these checks and cleaning up old code.